tests/test_export.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+import json
+from jsonschema import validate
+
+from tabgenie.utils.export import *
+
+'''-------------------JSON-SCHEMA-----------------------'''
+# https://python-jsonschema.readthedocs.io/en/latest/
+# need schema for testing tabs exporting into json
+with open('tests/schemes/schema_without_prop.json') as f:
+    schema_without_prop = json.load(f)
+
+with open('tests/schemes/schema_with_prop.json') as f:
+    schema_with_prop = json.load(f)
+'''-----------------------------------------------------'''
+
+
+def test_export_json(prepare_tests):
```

Comment:
please leave a comment __in the code__ about the argument name (that it's a fixture)